coalib/bearlib/languages/documentation/AssembledDocumentation.py

```diff
@@ -0,0 +1,71 @@
+from coalib.bearlib.languages.documentation.DocumentationComment import (
+    DocumentationComment)
+
+Description = DocumentationComment.Description
+Parameter = DocumentationComment.Parameter
+ReturnValue = DocumentationComment.ReturnValue
+
+
+def assemble_documentation(doccomment, param_sym, ret_sym):
+    """
+    Assembles a list of parsed documentation comments.
+
+    This function just assembles the documentation comment
+    itself, without the markers and indentation.
+
```

Comment:
a doctest would be really helpful, so the user sees how the format of the newly assembled documentation looks like :+1: